ops/manifests/manipulations.py

```diff
@@ -161,6 +162,7 @@ def __iter__(self) -> Iterator[AnyResource]:
             try:
                 return iter(obj)
             except TypeError:
+                obj = cast(AnyResource, obj)
                 return iter((obj,))
```

Comment:
this is testing if `obj` is iterable, if it's not -- use `cast` so the type system understand it's not Iterable